packages/components/ng-ui-router-layout/src/oui-modal/index.js

```diff
@@ -91,6 +109,10 @@ angular
             // create template
             const div = document.createElement('div');
             const elmt = document.createElement(kebabCase(directiveInfo.name));
+
+            // let pre = document.createElement('pre');
```

Comment:
Test purpose?

Theses lines could be removed I guess.

Reply:
good catch